.circleci/workflows.yml

```diff
@@ -83,28 +90,29 @@ jobs:
           at: /
 
       - restore_cache:
-          key: python-3.10.6-v1
+          key: python-3.10.6-v2
 
       - run: pyenv install --skip-existing 3.10.6
       - run: pyenv global 3.10.6
-      - run: pip install -U pip poetry
+      - run: pip install -U pip
+      - run: pip install poetry==1.2.2
 
       - restore_cache:
           keys:
-            - beamer-dependencies-v4-{{ checksum "poetry.lock" }}
-            - beamer-dependencies-v4
-      - run: poetry install
-      - run: echo "source $(poetry env info -p)/bin/activate" >> ~/.bashrc
+            - beamer-dependencies-v5-{{ checksum "poetry.lock" }}
+            - beamer-dependencies-v5
+      - run: poetry install -vvv
+      - run: grep -q activate ~/.bashrc || echo "source $(poetry env info -p)/bin/activate" >> ~/.bashrc
```

Comment:
Could we be more explicit here and check against `/bin/activate` for example?

Reply:
Done.